ra_aid/tools/write_file.py

```
@@ -51,14 +54,18 @@ def write_file_tool(
         logging.debug(f"Starting to write file: {filepath}")
 
         with open(filepath, "w", encoding=encoding) as f:
-            f.write(content)
-            result["bytes_written"] = len(content.encode(encoding))
+            logging.debug(f"Writing {len(complete_file_contents)} bytes to {filepath}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)